Fix Grammatical errors and sentence flows (#1684)
novuhq/novu: docs/docs/notification-center/iframe-embed.md

# iFrame Embed

> If you are using an (currently) unsupported client framework, you can use our embed script. This will generate the notification center inside an iframe.

Reviewer: ```suggestion
If you are using n (currently) unsupported client framework, you can use our embedded script. This will generate the notification center inside an iframe.
```

Author: This doesn't make sense

Reviewer: Sorry, fixed the mistyping.

> You can find the embed code in the `Settings` page, within the Admin Panel. It will look similar to this:

Reviewer: ```suggestion
You can find the embedded code in the `Settings` page, within the Admin Panel. It will look similar to this:
```

## Customizing the dropdown position

> Optionally, the embed init script receives a position object, you can use this to specify the `left` and `top` position of the notification center.

Reviewer: ```suggestion
Optionally, the embedded init script receives a position object, you can use this to specify the `left` and `top` position of the notification center.
```
